packages/devtools_app/lib/src/screens/profiler/cpu_profile_model.dart

```diff
@@ -449,22 +452,18 @@ class CpuProfileData {
   }
 }
 
-class CpuProfileMetaData {
+class CpuProfileMetaData extends ProfileMetaData {
   CpuProfileMetaData({
-    required this.sampleCount,
+    required int sampleCount,
```

Comment:
This would be a perfect place to use super initializers, but I guess we'll have to wait for the release to go stable... :-)